kani-driver/src/call_cbmc.rs

```diff
@@ -16,17 +16,35 @@ use crate::cbmc_output_parser::{
 use crate::cbmc_property_renderer::{format_result, kani_cbmc_output_filter};
 use crate::session::KaniSession;
 
-#[derive(Debug, PartialEq, Eq)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
 pub enum VerificationStatus {
     Success,
     Failure,
 }
 
+/// Classifies all possible outcomes when `#[kani::should_panic]` is being
+/// applied to the harness
+#[derive(Clone, Copy, Debug)]
+pub enum PanicOutcome {
```

Comment:
The `PanicOutcome` name gives an impression this only captures whether there were panics. Perhaps we can change this to:
```rust
pub enum FailedProperties {
    None,
    PanicsOnly,
    Other,
}
```